Cookbook/Cookbook/Recipes/MIDIPortTest.swift

```diff
@@ -0,0 +1,984 @@
+import AudioKit
+import AudioKitUI
+import CoreMIDI
+import Foundation
+import SwiftUI
+
+extension MIDIByte {
+
+    var controllerDescription: String {
```

Comment:
That's awesome! It would definitely save me time. I'll figure out how to use it later. I'm not sure how to implement it in this app yet since most of it is @ShowtimeSoft's code.

Reply:
could probably just copy this definition almost exactly straight over to that enum - no rush to do it but i bet that would work.

Reply:
Most of the values in the `controllerDescription` were redundant, so I just added a `description` property to your `MIDIControl.swift.`